src/Uno.UI.RuntimeTests/Tests/Windows_Storage/Streams/Given_DataWriter.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Windows.Storage.Streams;
+
+namespace Uno.UI.Tests.Windows_Storage.Streams
+{
+	[TestClass]
+	public class Given_DataWriter
+	{
+		[TestMethod]
+		public void Given_DateTime_MinValue()
+		{
+			var dataWriter = new DataWriter();
+			dataWriter.WriteDateTime(DateTimeOffset.MinValue);
+			var bytes = dataWriter.DetachBuffer().ToArray();
+			CollectionAssert.AreEqual(new byte[] { 248, 254, 49, 232, 221, 137, 0, 0 }, bytes);
+		}
+
+		[TestMethod]
+		public async Task When_WriteThenRead()
+		{
+			bool saveBool = true;
+			byte saveByte = 10;
+			int saveInt = 12345;
+			double saveDouble = 12345.678;
+			DateTimeOffset saveDate = DateTimeOffset.Now;
+			TimeSpan saveTimeSpan = TimeSpan.FromMinutes(10);
+			Guid saveGuid = Guid.NewGuid();
+
+			bool error = false;
+			var writer = new DataWriter();
+			try
+			{
+				writer.WriteBoolean(saveBool);
+				writer.WriteByte(saveByte);
+				writer.WriteInt32(saveInt);
+				writer.WriteDouble(saveDouble);
+				writer.WriteDateTime(saveDate);
+				writer.WriteTimeSpan(saveTimeSpan);
+				writer.WriteGuid(saveGuid);
+			}
+			catch
+			{
+				error = true;
+			}
+
+			if (error)
+			{
+				Assert.Fail("FAIL while writing (DataWriter)");
+			}
+
+			var buffer = writer.DetachBuffer();
+
+			var reader = DataReader.FromBuffer(buffer);
+
+			// variables below has to be initialized, because if not, we get CS0165 Use of unassigned local variable
+			// Codacy shows false errors here, it should be ignored.
+			bool loadBool = false;
+			byte loadByte = 0;
+			int loadInt = 0;
+			double loadDouble = 0;
+			DateTimeOffset loadDate = new DateTimeOffset(2000, 1, 1, 0, 0, 0, TimeSpan.FromSeconds(10));
+			TimeSpan loadTimeSpan = TimeSpan.FromSeconds(10);   // and saved value was 10 minutes
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'loadTimeSpan'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)